exclusions/.backupExclusionsEmpyrion1

```diff
@@ -0,0 +1,17 @@
+_CommonRedist
+Content
+DedicatedServer
+PlayfieldServer
+steamapps
+BuildNumber.txt
+dedicated.yaml
+Empyrion_Example.props
+EmpyrionDedicated.cmd
+EmpyrionDedicated_NoGraphics.cmd
+EmpyrionLauncher.exe
+steamclient.dll
+steamclient64.dll
+tier0_s.dll
+tier0_s64.dll
+vstdlib_s.dll
+vstdlib_s64.dll
```

Comment:
I will take your word for it that these are the appropriate files to exclude.

Reply:
Hopefully, it's correct. :)